prime-router/src/main/kotlin/cli/LookupTableCommands.kt

```diff
@@ -802,6 +837,14 @@ class LookupTableLoadAllCommand : GenericLookupTableCommand(
     )
         .flag(default = false)
 
+    /**
+     * Activate a created table in one shot.
+     */
+    private val forceTableToCreate by option(
+        "-f", "--force",
+        help = "Force the creation of new table(s) even if it is already exist"
```

Comment:
Fix the grammar here.
Force the creation of new table(s) even if it already exists

Reply:
Done. Thanks